src/components/omniSidebarNav/submenus/support.js

```diff
@@ -36,9 +36,9 @@ const support = () => {
         ]),
         simpleLink('/timeouts', 'Timeouts and Errors', [
           simpleLink('/timeouts', 'Timeouts on Pantheon'),
-          getGuideDirectory('guides/errors-and-server-responses'),
           simpleLink('/debug-connections', 'Debugging Connectivity Issues'),
         ]),
```

Comment:
@stevector @jazzsequence Does this layer still make sense without the multipage guide nested in it? Should the last 2 left here be moved up a layer as well?

First image shows the layer still intact w/2 left in it
<img width="349" alt="Screenshot 2025-01-23 at 2 57 12 PM" src="https://github.com/user-attachments/assets/82eb4128-6d8e-45c2-a5bb-68d6d8e24a8b" />

compared to this one, which moves the whole thing up a layer 
<img width="396" alt="Screenshot 2025-01-23 at 2 57 58 PM" src="https://github.com/user-attachments/assets/51d0221b-3144-45cf-bde0-d5ec45cbcdb7" />